From: Javier Viguera <javier.viguera@digi.com>
Date: Wed, 19 Apr 2017 14:04:51 +0200
Subject: [PATCH] port-probe: add more retries for port AT probing

Launching the MM on target's boot fails to detect the XBee cellular
because this needs more time to complete the initialization. So add some
more retries for the AT port probing to allow the XBee cellular to be
detected.

Upstream-Status: Inappropriate [embedded specific (Xbee cellular)]

Signed-off-by: Javier Viguera <javier.viguera@digi.com>
---
 src/mm-port-probe.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/mm-port-probe.c b/src/mm-port-probe.c
index cfe0763cfc3d..18cf53dd6b1d 100644
--- a/src/mm-port-probe.c
+++ b/src/mm-port-probe.c
@@ -1006,6 +1006,8 @@ static const MMPortProbeAtCommand at_probing[] = {
     { "AT",  3, mm_port_probe_response_processor_is_at },
     { "AT",  3, mm_port_probe_response_processor_is_at },
     { "AT",  3, mm_port_probe_response_processor_is_at },
+    { "AT", 15, mm_port_probe_response_processor_is_at },
+    { "AT", 20, mm_port_probe_response_processor_is_at },
     { NULL }
 };
 
